.github/workflows/linters.yml

```diff
@@ -69,6 +69,7 @@ jobs:
       - name: Format C/C++ files
         run: |
           make format-cpp-build
+          git branch -a
```

Comment:
actions/checkout only fetches a single commit by default. you need to either change the fetch-depth or call `git fetch` here

Reply:
Thanks for noticing, done!